tests/regressions/actions/plain_action_1330.cpp

```diff
@@ -18,7 +18,8 @@ namespace mynamespace
     HPX_DEFINE_PLAIN_ACTION(test, test_action);
 }
 
-HPX_REGISTER_PLAIN_ACTION(mynamespace::test_action, mynamespace_test_action);
+HPX_REGISTER_ACTION_DECLARATION(mynamespace::test_action, mynamespace_test_action);
+HPX_REGISTER_ACTION(mynamespace::test_action, mynamespace_test_action);
 
```

Comment:
Do we still need both macros? Or could we make it so that the second one would be sufficient?